Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -1584,6 +1584,26 @@ end
 					},
 					Notes = "Sends the specified animation of the specified entity to the client. The AnimationNumber is protocol-specific.",
 				},
+				SendPlayerListHeaderFooter =
+				{
+					{
+						Params =
+						{
+							{
+								Type = "cCompositeChat",
+								Name = "a_Header",
+							},
+							{
+								Type = "cCompositeChat",
+								Name = "a_Footer",
+							},
+						},
+						Returns =
+						{
+						},
+						Desc = "Sends the header and footer of the player list to the client."
+					},
+				},
```

Comment:
Looks like an extra set of outer braces, and an extra Returns. Could you also delete the a_ from a_Header/a_Footer, and the same for the other API additions.